src/main/java/model/Player.java

```diff
@@ -0,0 +1,45 @@
+package model;
+
+
+import exception.InvalidMoneyAmountException;
+
+public class Player extends Person{
+    private int wallet;
+    private int bet;
+    private int playerNumber;
+
+    public Player(){
+        wallet = 100;
+    }
+
+    /**
+     * @return the balance of the player's wallet
+     */
+    public int balance(){
+        return this.wallet;
+    }
+
+    /**
+     * withdraws a certain amount of money from the player's wallet
+     */
+    public void withdraw(int amount) throws InvalidMoneyAmountException {
+        if (this.wallet < amount || amount == 0) {
```

Comment:
I would suggest to differ between withdraw more money than there is money in the wallet and withdraw 0 (or less) money.
So I would suggest to add two exceptions: InvalidMoneyAmount (for an amount <=0) and an exception like NotEnoughMoneyAmount for an amount higher than the amount in the wallet.